### PR TITLE
mystery solved

### PR DESCRIPTION
**Pull request by poncho678**

### CODE REVIEW

File: src/clue.js [hunk 1]
```diff
@@ -1,30 +1,128 @@
...
+function selectRandom(arr) {
+  if (!arr.length) {
+    return undefined;
+  }
+  const randomNumber = Math.floor(Math.random() * arr.length);
+  return arr[randomNumber];
+}
 
+function pickMystery() {
+  const mysteryObject = {};
```
Comment (filipe-freire) [inline src/clue.js:109]: Could've also declared the mystery object like this:

```
const mystery = {
    suspect: selectRandom(suspectsArray),
    weapon: selectRandom(weaponsArray),
    room: selectRandom(roomsArray)
}
```

File: src/clue.js [hunk 2]
```diff
@@ -1,30 +1,128 @@
...
 
-function revealMystery() {}
-
-
+function revealMystery(cards) {
+  const {
+    suspect: { firstName, lastName },
+    weapon: { name: weapon },
+    room: { name: room }
+  } = cards;
```
Comment (filipe-freire) [inline src/clue.js:122]: Uh, advanced destructuring right here! 👀 🥇

File: src/clue.js (hunk 1 above)
Comment (poncho678) [reply src/clue.js:109]: Ok cool, this is a lot shorter. Thank you

File: src/clue.js (hunk 2 above)
Comment (filipe-freire) [reply src/clue.js:122]: Definitely will be covered! 💪🏻

**Pull request closed without merging**